src/qtoolkit/io/pbs.py

```diff
@@ -120,6 +120,10 @@ def _get_qstat_base_command(self) -> list[str]:
     def _get_job_ids_flag(self, job_ids_str: str) -> str:
         return job_ids_str
 
+    def _get_job_cmd(self, job_id: str):
+        cmd = f"qstat -f {job_id}"
+        return cmd
```

Comment:
for PBS, I have restored the previous state of this function, but I wonder if we want to switch to xml output like that
```suggestion
    def _get_job_cmd(self, job_id: str):
        cmd = f"qstat -fx {job_id}"
        return cmd
```
maybe?

Reply:
Ok, then I will leave it like that!